src/lib/lib.c

```diff
@@ -114,10 +114,15 @@ int rstrcmp(const char *s1, const char *s2)
 		if (*s1 < *s2)
 			return -1;
 		if (*s1 > *s2)
-			return 1;
+			return  1;
```

Comment:
why add a space here.

Reply:
For human readability purposes. Is this against style guidelines of this project?

Reply:
@lgirdwood @plbossart Is this an issue?

Reply:
This creates differences for no good reason. We typically try to stick to existing style/alignment unless there is a visible improvement, which isn't the case there.

Reply:
@plbossart Acknowledged. Will change.